src/Playback/__tests__/Playback.test.jsx

```diff
@@ -0,0 +1,133 @@
+import "@testing-library/jest-dom";
+import { render, screen, fireEvent } from "@testing-library/react";
+import { describe, it, expect, beforeEach, vi } from "vitest";
+
+import ChannelPlayer from "@/Playback/components/ChannelPlayer";
+import useChannelPlayback from "@/Playback/hooks/useChannelPlayback";
+import useUpdateSetting from "@/User/hooks/useUpdateSetting";
+import { useUserStore } from "@/User/stores/useUserStore";
+
+let mockIsChannelChanged = false;
+let mockIsPlaying = false;
+
+vi.mock("@/Channel/stores/useChannelStore", () => ({
+  useChannelStore: (selector) =>
+    selector({
+      isChannelChanged: mockIsChannelChanged,
+    }),
+}));
+
+vi.mock("@/User/stores/useUserStore", () => ({
+  useUserStore: vi.fn(),
+}));
+
+vi.mock("@/Playback/hooks/useChannelPlayback", () => ({
+  default: vi.fn(),
+}));
+
+vi.mock("@/User/hooks/useUpdateSetting", () => ({
+  default: vi.fn(),
+}));
+
+vi.mock("@/User/constants/settingOptions", async (importOriginal) => {
+  const actual = await importOriginal();
+  return {
+    ...actual,
+    SETTING_TITLES: {
+      ...actual.SETTING_TITLES,
+      AD_DETECT: "광고 감지",
+      RETURN_CHANNEL: "기존 채널로 이동",
+    },
+  };
+});
+
+describe("ChannelPlayer", () => {
+  const mockUpdateSetting = vi.fn();
+  const mockHandlePlayPause = vi.fn();
+
+  beforeEach(() => {
+    vi.clearAllMocks();
+    useUserStore.mockReturnValue({
+      settings: {
+        RETURN_CHANNEL: true,
+        AD_DETECT: true,
+      },
+    });
+
+    useChannelPlayback.mockReturnValue({
+      selectedChannel: {
+        name: "테스트 채널",
+        logoUrl: "https://example.com/test.jpg",
+      },
```

Comment:
```suggestion
  const mockSelectedChannel = {
    name: "테스트 채널",
    logoUrl: "https://example.com/test.jpg",
  };
```

`selectedChannel mock` 객체가 동일하게 반복되고 있어서, 공통 `mock` 객체로 분리하고 `selectedChannel: mockSelectedChannel`형태로 사용하면 가독성도 좋아지고 유지보수도 더 편할 것 같아요!

Reply:
좋은 리뷰 감사합니다!
말씀해주신 부분 반영해서 공통 `mock` 객체로 분리해보겠습니다 :)
가독성도 좋아지고 유지보수도 확실히 편해질 것 같아요!